utility/src/main/java/org/oppia/util/firebase/CrashlyticsWrapper.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for providing custom crash reporting to Firebase Crashlytics */
+@Singleton
+open class CrashlyticsWrapper @Inject constructor(
+  private var firebaseCrashlytics: FirebaseCrashlytics
+) {
+
+  /** Logs a custom non-fatal exception to Firebase Crashlytics */
```

Comment:
In general each comment statement should have `.` at the end. Ditto elsewhere in this file.

Reply:
Done